driver/normalizer/fixtures/var.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
This kind of fixture should be in the `test` directory, so that it can be used in integration tests.